### PR TITLE
fix to support multiple ble dongle and to use 21-digits paring code

### PR DESCRIPTION
**Pull request by dooho-lee**
1. fix to support multiple ble dongle 
2. fix to use 21-digits pairing code instead of 11-digits pairing code.
- in case of 11-digits code, duplication can occur and another device can be onboarded.
- ioter-stable commit ([aeaa2b6431b1893dbc02b81536a725010862d15d](https://github.ecodesamsung.com/THREAD/ioter/commit/aeaa2b6431b1893dbc02b81536a725010862d15d))

### CODE REVIEW

File: script/start [hunk 1]
```diff
@@ -76,7 +76,7 @@ ioter_restart() {
 }
 
 main() {
-    bluetooth_restart
+    #bluetooth_restart
```
Comment (junyong-sim) [inline script/start:79]: Can we move this function to `script/run` ?

If user face some issues related to bluetooth, they might need to restart bluetooth.
If we move it to `script/run`, use would solve bluetooth issue by just restarting ioter app.

Comment (dooho-lee) [reply script/start]: > Can we move this function to `script/run` ?
> 
> If user face some issues related to bluetooth, they might need to restart bluetooth. If we move it to `script/run`, use would solve bluetooth issue by just restarting ioter app.

How about now? I fixed codes.
bluetooth_restart function is moved so bluetooth restart once whenever start main.py

**Review by junyong-sim — APPROVED** [review]
LGTM

**Review by spdkimo — APPROVED** [review]
LGTM

**Pull request merged**